package.json

```diff
@@ -15,7 +15,8 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.20",
+    "@docusaurus/core": "2.0.0-beta.21",
```

Comment:
Local build kept complaining that these 2 need to be the same version. Didn't know how to do so while using ^ notation